src/utils/helpers/date/date.js

```diff
@@ -206,38 +156,30 @@ const DateHelper = {
    * @param {Object} options Override Moment JS options
    * @return {Moment}
    */
-  _parseDate(value, options) {
-    const opts = merge(DateHelper._defaultMomentOptions(), options);
-    const val = opts.sanitize ? DateHelper.sanitizeDateInput(value) : value;
-    return moment(
-      val,
-      [this._visibleFormat(), ...opts.formats],
-      opts.locale,
-      opts.strict
+  _parseDate({ value, options, locale, formats, format }) {
+    const opts = merge(
+      DateHelper._defaultMomentOptions(locale, formats),
+      options,
+      {
+        locale,
+        formats,
+        format,
+      }
     );
+    const val = opts.sanitize ? DateHelper.sanitizeDateInput(value) : value;
+    return moment(val, [format, ...opts.formats], opts.locale, opts.strict);
   },
 
-  /**
-   * Formats valid for entry
-   *
-   * @private
-   * @method validFormats
-   * @return {Array} formatted date strings
-   */
-  _dateFormats: () => {
-    return I18n.t("date.formats.inputs", {
-      defaultValue: DateHelper._defaultDateFormats(),
-    });
-  },
-
-  _visibleFormat: () =>
-    I18n.t("date.formats.javascript", {
-      defaultValue: defaultDateFormat,
-    }).toUpperCase(),
-
-  formatDateToCurrentLocale(value) {
```

Comment:
Please document this as a breaking change.

Reply:
added, thanks